test/int/integration_suite_test.go

```diff
@@ -199,7 +203,11 @@ func prepareControllers(deletionProtection bool) (*corev1.Namespace, context.Can
 	// Note on the syncPeriod - decreasing this to a smaller time allows to test its work for the long-running tests
 	// (deployments, database users). The prod value is much higher
 	syncPeriod := time.Minute * 30
-	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
+
+	// shallow copy global config
+	managerCfg := *cfg
+	managerCfg.UserAgent = "AKO"
+	k8sManager, err := ctrl.NewManager(&managerCfg, ctrl.Options{
```

Comment:
Note: these changes will be removed in this PR (and contributed separately). I want them here to debug audit logs in case of e2e failures.

Reply:
leaving those changes in this PR as we had no opposition to remove them, adding them to the description.